apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -33,7 +33,9 @@ function LoginPage(props: LoginPageProps) {
             </div>
             <StandaloneLoginForm
               sessionCallback={() => {
-                redirectAfterSSO(returnToUrl)
+                redirectAfterSSO(history, returnToUrl)
+                // If we didn't redirect, refresh the session
+                refreshSession()
```

Comment:
I changed the redirect behavior to not necessarily refresh the page if we can just use `react-router` to redirect. Since may not do a full page reload, we need to refresh the login session.